miranda/structure/_structure.py

```diff
@@ -78,89 +78,75 @@ def build_path_from_schema(
+                "variable",
             )
 
         if facets["type"] == "simulation":
             SIMULATION_SCHEMA.validate(facets)
             if facets["processing_level"] == "raw":
-                try:
-                    if facets["project"] == "CORDEX":
-                        return (
-                            Path(output_folder)
-                            / facets["type"]
-                            / facets["processing_level"]
-                            / facets["activity"]
-                            / facets["mip_era"]
-                            / facets["project"]
-                            / facets["domain"]
-                            / facets["source"]
-                            / facets["driving_model"]
-                            / facets["experiment"]
-                            / facets["member"]
-                            / facets["frequency"]
-                            / facets["variable"]
-                        )
-                except KeyError:
-                    return (
-                        Path(output_folder)
-                        / facets["type"]
-                        / facets["processing_level"]
-                        / facets["activity"]
-                        / facets["mip_era"]
-                        / facets["domain"]
-                        / facets["institution"]
-                        / facets["source"]
-                        / facets["experiment"]
-                        / facets["member"]
-                        / facets["frequency"]
-                        / facets["variable"]
-                    )
+                folder_tree_structure = (
+                    "type",
+                    "processing_level",
+                    "mip_era",
+                    "activity",
+                    "project",
+                    "domain",
+                    "source",
+                    "driving_model" if facets["project"] == "CORDEX" else None,
+                    "experiment",
+                    "member",
+                    "frequency",
+                    "variable",
+                )
+
             elif facets["processing_level"] == "biasadjusted":
-                return (
-                    Path(output_folder)
-                    / facets["type"]
-                    / facets["processing_level"]
-                    / facets["activity"]
-                    / facets["mip_era"]
-                    / facets["bias_adjust_institution"]
-                    / facets["bias_adjust_project"]
-                    / facets["domain"]
-                    / facets["institution"]
-                    / facets["source"]
-                    / facets["experiment"]
-                    / facets["member"]
-                    / facets["frequency"]
-                    / facets["variable"]
+                folder_tree_structure = (
+                    "type",
+                    "processing_level",
+                    "activity",
```

Comment:
```suggestion
```
I don't think we need activty for bias_adjusted. It is all defined by the `bias_adjuste_project`. For example, ESPO-R5v1 uses CORDEX by definition. ESPO-G6v1 use CMIP(for historical) and ScenarioMIP.

Reply:
If we absolutely want to keep it, it should at least be after `mip_era`.